### PR TITLE
docs: fix broken admonitions

### PR DESCRIPTION
**Pull request by rickstaa**
This pull request ensures that the admonitions are in line with the new GFM specifications (see https://github.com/orgs/community/discussions/16925#discussioncomment-6506860).

### CODE REVIEW

**Review by qwerty541 — COMMENTED** [review]
@rickstaa Nice improvement, but i see there a lot of other changes like replacing `-` symbols with `*` in all kinds of lists and backslashes before all `&`, `[` and `]` symbols. I assume that it was not manual chacnges, if we're going to use it, workspace settings should be updated. Otherwise if i will do some changes in this file in future and press `Ctrl + S`, these changes most likely will be reverted.

**Comment by rickstaa** [comment]
> @rickstaa Nice improvement, but i see there a lot of other changes like replacing `-` symbols with `*` in all kinds of lists and backslashes before all `&`, `[` and `]` symbols. I assume that it was not manual chacnges, if we're going to use it, workspace settings should be updated. Otherwise if i will do some changes in this file in future and press `Ctrl + S`, these changes most likely will be reverted.

These changes were added by Vscode when I saved the file using `Ctrl + S` so assumed they were the ones you put in the config file 🤔. Strange.

**Comment by qwerty541** [comment]
> > @rickstaa Nice improvement, but i see there a lot of other changes like replacing `-` symbols with `*` in all kinds of lists and backslashes before all `&`, `[` and `]` symbols. I assume that it was not manual chacnges, if we're going to use it, workspace settings should be updated. Otherwise if i will do some changes in this file in future and press `Ctrl + S`, these changes most likely will be reverted.
> 
> These changes were added by Vscode when I saved the file using `Ctrl + S` so assumed they were the ones you put in the config file 🤔. Strange.

We have only one config related to markdown: https://github.com/anuraghazra/github-readme-stats/blob/a3b8a7856cb8a861a3267286ea76c80894279007/.vscode/settings.json#L2

Most likely you have some configs on user's level, you should move them into workspace configs or discard changes.

**Comment by rickstaa** [comment]
> > > @rickstaa Nice improvement, but I see there a lot of other changes like replacing `-` symbols with `*` in all kinds of lists and backslashes before all `&`, `[` and `]` symbols. I assume that it was not manual chacnges, if we're going to use it, workspace settings should be updated. Otherwise if i will do some changes in this file in future and press `Ctrl + S`, these changes most likely will be reverted.
> > 
> > 
> > These changes were added by Vscode when I saved the file using `Ctrl + S` so assumed they were the ones you put in the config file thinking. Strange.
> 
> We have only one config related to markdown:
> 
> https://github.com/anuraghazra/github-readme-stats/blob/a3b8a7856cb8a861a3267286ea76c80894279007/.vscode/settings.json#L2
> 
> Most likely you have some configs on user's level, you should move them into workspace configs or discard changes.

We need to choose a default format for both JS and markdown 👍🏻. I have three on my system. Which one do you want to use for markdown? Also prettier?

**Comment by qwerty541** [comment]
> > > > @rickstaa Nice improvement, but I see there a lot of other changes like replacing `-` symbols with `*` in all kinds of lists and backslashes before all `&`, `[` and `]` symbols. I assume that it was not manual chacnges, if we're going to use it, workspace settings should be updated. Otherwise if i will do some changes in this file in future and press `Ctrl + S`, these changes most likely will be reverted.
> > > 
> > > 
> > > These changes were added by Vscode when I saved the file using `Ctrl + S` so assumed they were the ones you put in the config file thinking. Strange.
> > 
> > 
> > We have only one config related to markdown:
> > https://github.com/anuraghazra/github-readme-stats/blob/a3b8a7856cb8a861a3267286ea76c80894279007/.vscode/settings.json#L2
> > 
> > Most likely you have some configs on user's level, you should move them into workspace configs or discard changes.
> 
> We need to choose a default format for both JS and markdown 👍🏻. I have three on my system. Which one do you want to use for markdown? Also prettier?

Does `yzhang.markdown-all-in-one` extension provide formatter for markdown as well? We can keep prettier as formatter for JS and set another formatter for markdown if it is better at something.

**Comment by rickstaa** [comment]
> > > > > @rickstaa Nice improvement, but I see there a lot of other changes like replacing `-` symbols with `*` in all kinds of lists and backslashes before all `&`, `[` and `]` symbols. I assume that it was not manual chacnges, if we're going to use it, workspace settings should be updated. Otherwise if i will do some changes in this file in future and press `Ctrl + S`, these changes most likely will be reverted.
> > > > 
> > > > 
> > > > These changes were added by Vscode when I saved the file using `Ctrl + S` so assumed they were the ones you put in the config file thinking. Strange.
> > > 
> > > 
> > > We have only one config related to markdown:
> > > https://github.com/anuraghazra/github-readme-stats/blob/a3b8a7856cb8a861a3267286ea76c80894279007/.vscode/settings.json#L2
> > > 
> > > Most likely you have some configs on user's level, you should move them into workspace configs or discard changes.
> > 
> > 
> > We need to choose a default format for both JS and markdown 👍🏻. I have three on my system. Which one do you want to use for markdown? Also prettier?
> 
> Does `yzhang.markdown-all-in-one` extension provide formatter for markdown as well? We can keep prettier as formatter for JS and set another formatter for markdown if it is better at something.

Yea, the incorrect formatting came from that extension. I now updated the `settings.json` to prevent that from happening. Let's stay with Prettier since it is the most used and most versatile it also is the one we already use.

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
     "markdown.extension.toc.levels": "1..3",
     "editor.formatOnSave": true,
-    "editor.defaultFormatter": "esbenp.prettier-vscode" 
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "[markdown]": {
+        "editor.defaultFormatter": "esbenp.prettier-vscode"
+    }
```
Comment (qwerty541) [inline .vscode/settings.json:7]: Does `editor.defaultFormatter` option affect on the whole workspace including all file extensions? I'm not sure if we need to define same option for markdown files separately.

**Review by qwerty541 — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (qwerty541) [reply .vscode/settings.json]: @rickstaa I resolved this

**Pull request merged**